python/res/enkf/local_dataset.py

```diff
@@ -39,6 +42,12 @@ def __delitem__(self, key):
         else:
             raise KeyError('Unknown key "%s"' % key)
 
+    def rowScaling(self, key):
+        if not key in self:
```

Comment:
`key not in self` rather than `not key in self`

Reply:
OK